pkg/proc/native/proc_darwin.go

```diff
@@ -446,6 +426,38 @@ func (dbp *Process) resume() error {
 	return nil
 }
 
+// stop stops all running threads threads and sets breakpoints
+func (dbp *Process) stop(trapthread *Thread) (err error) {
+	if dbp.exited {
+		return &proc.ProcessExitedError{Pid: dbp.Pid()}
+	}
+	for _, th := range dbp.threads {
+		if !th.Stopped() {
+			if err := th.halt(); err != nil {
```

Comment:
nit: Let's be consistent with naming and call both `stop` or `halt`. I'm leaning towards `halt` because we also have that flag on the OS struct as well.

Reply:
It was called halt before but it did a different thing, given that the meaning changed the name should change too, IMHO.

Reply:
Are you in favor of switching consistently to `stop`?

Reply:
Done.